datastore/mongo/subscription.go

```diff
@@ -152,3 +152,20 @@ func (s *subscriptionRepo) FindSubscriptionBySourceIDs(ctx context.Context, grou
 
 	return subscription, nil
 }
+
+func (s *subscriptionRepo) UpdateSubscriptionsStatus(ctx context.Context, groupId string, subscriptionId string, status datastore.EndpointStatus) error {
```

Comment:
I think this should be `UpdateSubscriptionStatus`

Reply:
Ops, updated it. Thanks

Reply:
Nice